tox.ini

```diff
@@ -16,10 +16,7 @@ usedevelop = True
 commands = {posargs}
 
 [testenv:pep8]
-commands = flake8 {posargs}
-
-[flake8]
-exclude=.tox
```

Comment:
I am not so convinced about this one, I think the `{posargs}` have no real sense in `pep8` venv, but let it be...